packages/create-cloudflare/templates/next/wrangler.toml

```diff
@@ -53,3 +54,24 @@ compatibility_flags = ["nodejs_compat"]
 # binding = "MY_SERVICE"
 # service = "my-service"
 
+# To use different bindings for preview and production environments, follow the examples below.
+# When using environment-specific overrides for bindings, ALL bindings must be specified on a per-environment basis.
+# Docs: https://developers.cloudflare.com/pages/functions/wrangler-configuration#environment-specific-overrides
+
+######## PREVIEW environment config ########
+
+# [env.preview.vars]
+# API_KEY = "xyz789"
+
+# [[env.preview.kv_namespaces]]
+# binding = "MY_KV_NAMESPACE"
+# id = "<PREVIEW_NAMESPACE_ID>"
```

Comment:
Everywhere else we present examples of KVs, as their id we use: `# id = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"`
I don't like that this is inconsistent with that 😅 

I can see why you're doing it (to distinguish between preview and prod ids)....

I don't think this needs to block this PR, but I would suggest to maybe, as a followup replace all the `xxxx` values we've been using with this sort of thing? (e.g.`<NAMESPACE_ID>`, `<DATABASE_ID>`, etc...)

Reply:
This doesn't really bug me, and I think there's value in it. For example, the placeholder we use for D1 ids is: `"xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"`, which gives the user a little extra affordance that what they're copying is the correct thing.

It's ok to be inconsistent at times when there's a good reason for it 😄.